README.md

```diff
@@ -121,14 +154,14 @@ kubectl create secret generic kbs-auth-public-key --from-file=kbs.pem -n kbs-ope
 5. Create the KBS and AS configmaps
 
 ```sh
-kubectl apply -f config/samples/kbs-config.yaml
-kubectl apply -f config/samples/as-config.yaml
+kubectl apply -f config/samples/kbs-config-grpc.yaml
```

Comment:
Instead you can have two sub-dirs under samples for both the deployment types. And refer to them in the README